composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "eastwest/json",
-    "type": "library",
+    "type": "package",
```

Comment:
the `type` in `composer.json` should either remain at `library`, or be dropped entirely (it defaults to `library`). `package` is a made-up name that has no official support in Composer and possibly third-party plugins.

Reply:
Good catch.

Basically it does nothing and can be, indeed, omitted. 

For reference 
https://getcomposer.org/doc/04-schema.md#type